src/improvements/tasks/eth/007-opcm-update-prestate-v300-op+ink/config.toml

```diff
@@ -0,0 +1,47 @@
+templateName = "OPCMUpdatePrestateV300"
+
+[[l2chains]]
+chainId = 10
+name = "OP Mainnet"
+
+[[l2chains]]
+chainId = 57073
+name = "Ink"
+
+
+[[opcmUpgrades]]
+chainId = 10
+# https://www.notion.so/oplabs/Isthmus-Sepolia-Mainnet-1d2f153ee162800880abe1b47910c071
+absolutePrestate = "0x03682932cec7ce0a3874b19675a6bbc923054a7b321efc7d3835187b172494b6"
+# PDDG-DWETH-30: Permissioned dispute game's DelayedWETH owner must be l1PAOMultisig
+# PDDG-ANCHORP-40: Permissioned dispute game's AnchorStateRegistry root must be 0xdead000000000000000000000000000000000000000000000000000000000000
+# PLDG-DWETH-30: Permissionless dispute game's DelayedWETH owner must be l1PAOMultisig
+# PLDG-ANCHORP-40: Permissionless dispute game's AnchorStateRegistry root must be 0xdead000000000000000000000000000000000000000000000000000000000000
+expectedValidationErrors = "PDDG-DWETH-30,PDDG-ANCHORP-40,PLDG-DWETH-30,PLDG-ANCHORP-40"
+
+[[opcmUpgrades]]
+chainId = 57073
+# https://www.notion.so/oplabs/Isthmus-Sepolia-Mainnet-1d2f153ee162800880abe1b47910c071
+absolutePrestate = "0x03682932cec7ce0a3874b19675a6bbc923054a7b321efc7d3835187b172494b6"
+# SYSCON-30: System config scalar must be 1 << 248 (first byte must be 1) - SYSCON-30 is a result of the Ink system config being on an earlier.
```

Comment:
Looks like the sentence is truncated here, curious to better understand the reason for this error

Reply:
I copied this from an earlier task